src/ajax.js

```diff
@@ -711,7 +711,7 @@ jQuery.extend( {
 		}
 
 		// Callback for when everything is done
-		function done( status, nativeStatusText, responses, headers ) {
+		function done( status, nativeStatusText, responses, headers, responseURL ) {
```

Comment:
Unfortunately, this function signature is public API as that's the shape of `completeCallback` as described here: https://api.jquery.com/jQuery.ajaxTransport/

This is a new parameter at the end so it's not a breaking change. However, we need to think if this is a good API as 5 unnamed parameters is quite a lot.

(from my comment from https://github.com/jquery/jquery/pull/4405#discussion_r305066924)

Reply:
FYI: I opened #4634 to make adding `responseURL` possible without adding more parameters here.